cursor.go

```diff
@@ -261,6 +261,7 @@ func (c *Cursor) Fetch(db SqlPreparer) (batch *RowBatch, paginationKeypos uint64
 		values:             batchData,
 		paginationKeyIndex: paginationKeyIndex,
 		table:              c.Table,
+		columns:            columns,
```

Comment:
I was thinking a bit about it and thought it was not worth the extra compute time since this is a critical flow for performance. Inline verifier would catch if these are different.

Reply:
I guess I feel pretty paranoid, but you're right the InlineVerifier should catch this so this is probably fine.. 

A note about performance: the RowBatch inserts should be done with hundreds to thousands of rows at the same time. My last measurement of this show that bulk of the time is spent waiting on MySQL as opposed to be within Ghostferry (something like 90%+ time is spent in MySQL). I would imagine checking the columns once per batch is going to be negligible in terms of compute cost.